src/java.management/share/classes/javax/management/modelmbean/DescriptorSupport.java

```diff
@@ -967,7 +974,10 @@ private boolean validateField(String fldName, Object fldValue) {
      * field Names or field Values.  If the XML formatted string
      * construction fails for any reason, this exception will be
      * thrown.
+     * @deprecated This method exists for historical reasons.  If
+     * reading from XML is required, it should be implemented externally.
```

Comment:
Technically here it should be `writing to XML` - I don't think changing that would require re-submitting the CSR - but maybe @jddarcy could confirm?